documentation/src/main/asciidoc/introduction/Entities.adoc

```diff
@@ -649,7 +649,7 @@ There are two ways to apply a converter:
 - the `@Convert` annotation applies an `AttributeConverter` to a particular entity attribute, or
 - the `@Converter` annotation (or, alternatively, the `@ConverterRegistration` annotation) registers an `AttributeConverter` for automatic application to all attributes of a given type.
 
-For example, the following converter will be automatically applied to any attribute of type `BitSet`, and takes care of persisting the `BitSet` to a column of type `varbinary`:
+For example, the following converter will be automatically applied to any attribute of type `EnumSet<DayOfWeek>`, and takes care of persisting the `EnumSet<DayOfWeek>` to a column of type `INTEGER`:
 
```

Comment:
What was wrong with using `BitSet` in this example?

Reply:
<img width="1096" alt="Screenshot 2024-10-03 at 08 40 00" src="https://github.com/user-attachments/assets/840957bd-2195-4a88-bf34-6f3c31fe2fe0">
Here is the context. It is confusing, for the Converter example just below is not using `BitSet` but `EnumSet<DayOfWeek>`. Then after this example class, `BitSet` is used again. 
I am trying to make both consistent. Using `EnumSet` instead of `BitSet` could avoid rewriting that converter class.

Reply:
Oh, yeah, sure, you're right.